index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Resume Builder</title>
+  <script src="app.js"></script>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <form id="resume-form" method="post">
+    <div>
```

Comment:
There is an opportunity here to use DRY principles here and generate these input fields using javascript instead. You could create an array of Label names and loop through it to generate the form inputs instead.